lib/functions/settings/RenderSettings.lua

```diff
@@ -0,0 +1,14 @@
+local RenderSettings = {}
+RenderSettings.__index = RenderSettings
+RenderSettings.type = "RenderSettings"
+
+function RenderSettings.new()
+	local instance = {
+		QualityLevel = 0,
+	}
+
+	setmetatable(instance, RenderSettings)
```

Comment:
Same here as in `Settings` -- I think we should make this a `userdata` with fairly strict access so that the rest of Lemur can make stronger assumptions about these kinds of objects.